kaggle.json

```diff
@@ -0,0 +1 @@
+{"username":"<place_your_username_here>","key":"<place_your_key_here>"}
```

Comment:
Could be context specific, like `your_kaggle_username` or `place_your_kaggle_username_here` (littlebit long).

Reply:
Replaced with specific text.